client/attest.go

```diff
@@ -236,8 +239,10 @@ func (k *Key) Attest(opts AttestOpts) (*pb.Attestation, error) {
 		}
 		attestation.Quotes = append(attestation.Quotes, quote)
 	}
-	if attestation.EventLog, err = GetEventLog(k.rw); err != nil {
-		return nil, fmt.Errorf("failed to retrieve TCG Event Log: %w", err)
+	if opts.TCGEventLog == nil {
```

Comment:
This option appears only ever to be written and never read. Wouldn't the opts.TCGEventLog != nil case need to have attestation.EventLog = opts.TCGEventLog?

Reply:
Yes , this check should be there otherwise `attestation.EventLog` is always nil while testing.
Thank you .

Reply:
I have implemented that .